parquet-cli/src/test/resources/cli-output/show_footer_command.txt

```diff
@@ -0,0 +1,331 @@
+      "valueCount" : -2147483638,
+      "totalSize" : -2147483610,
+      "totalUncompressedSize" : -2147483610
+    }, {
+      "rowGroupOrdinal" : 0,
+      "encodingStats" : {
+        "dictStats" : { },
+        "dataStats" : {
+          "DELTA_BINARY_PACKED" : 1
+        },
+        "usesV2Pages" : true
+      },
+      "properties" : {
+        "codec" : "UNCOMPRESSED",
+        "path" : {
+          "p" : [ "int64_field" ]
+        },
+        "encodings" : [ "DELTA_BINARY_PACKED" ]
+      },
+      "columnIndexReference" : {
+        "offset" : 596,
+        "length" : 31
+      },
+      "offsetIndexReference" : {
+        "offset" : 778,
+        "length" : 10
+      },
+      "bloomFilterOffset" : -1,
+      "bloomFilterLength" : -1,
+      "firstDataPage" : -2147483606,
+      "dictionaryPageOffset" : -2147483648,
+      "valueCount" : -2147483638,
+      "totalSize" : -2147483610,
+      "totalUncompressedSize" : -2147483610
+    }, {
+      "rowGroupOrdinal" : 0,
+      "encodingStats" : {
+        "dictStats" : { },
+        "dataStats" : {
+          "PLAIN" : 1
+        },
+        "usesV2Pages" : true
+      },
+      "properties" : {
+        "codec" : "UNCOMPRESSED",
+        "path" : {
+          "p" : [ "float_field" ]
+        },
+        "encodings" : [ "PLAIN" ]
+      },
+      "columnIndexReference" : {
+        "offset" : 627,
+        "length" : 23
+      },
+      "offsetIndexReference" : {
+        "offset" : 788,
+        "length" : 12
+      },
+      "bloomFilterOffset" : -1,
+      "bloomFilterLength" : -1,
+      "firstDataPage" : -2147483568,
+      "dictionaryPageOffset" : -2147483648,
+      "valueCount" : -2147483638,
+      "totalSize" : -2147483580,
+      "totalUncompressedSize" : -2147483580
+    }, {
+      "rowGroupOrdinal" : 0,
+      "encodingStats" : {
+        "dictStats" : { },
+        "dataStats" : {
+          "PLAIN" : 1
+        },
+        "usesV2Pages" : true
+      },
+      "properties" : {
+        "codec" : "UNCOMPRESSED",
+        "path" : {
+          "p" : [ "double_field" ]
+        },
+        "encodings" : [ "PLAIN" ]
+      },
+      "columnIndexReference" : {
+        "offset" : 650,
+        "length" : 31
+      },
+      "offsetIndexReference" : {
+        "offset" : 800,
+        "length" : 12
+      },
+      "bloomFilterOffset" : -1,
+      "bloomFilterLength" : -1,
+      "firstDataPage" : -2147483500,
+      "dictionaryPageOffset" : -2147483648,
+      "valueCount" : -2147483638,
+      "totalSize" : -2147483538,
+      "totalUncompressedSize" : -2147483538
+    }, {
+      "rowGroupOrdinal" : 0,
+      "encodingStats" : {
+        "dictStats" : {
+          "PLAIN" : 1
+        },
+        "dataStats" : {
+          "RLE_DICTIONARY" : 1
+        },
+        "usesV2Pages" : true
+      },
+      "properties" : {
+        "codec" : "UNCOMPRESSED",
+        "path" : {
+          "p" : [ "binary_field" ]
+        },
+        "encodings" : [ "PLAIN", "RLE_DICTIONARY" ]
```

Comment:
it's a HashSet, the output is unstable, see https://github.com/FasterXML/jackson-databind/issues/3166